tools/benchmark/macros/Cargo.toml

```diff
@@ -11,5 +11,10 @@ version = "0.1.0"
 [dependencies]
 dhat = { version = "0.2", optional = true }
 
+# This cfg originates in dlmalloc/lib.rs
+[target.'cfg(any(target_os = "linux", target_os = "macos", target_arch = "wasm32"))'.dependencies]
+dlmalloc = { version = "0.2", optional = true, features = ["global"] }
+
 [features]
 benchmark_memory = ["dhat"]
+rust_global_allocator = ["dlmalloc"]
```

Comment:
Suggestion: This is a new feature. We have #556 on file to document the features, but it would be good to either add it to that issue to follow-up, or stub out the feature documentation with this feature.

Reply:
Done.